ixwebsocket/IXSocketOpenSSL.cpp

```diff
@@ -283,17 +327,19 @@ namespace ix
             }
             SSL_set_fd(_ssl_connection, _sockfd);
 
```

Comment:
does SNI depends on peer validation ?

Reply:
nope :) I was thinking of the security aspects, but the https->virtual host aspects would still apply; removed